### ISSUE TITLE
Need help for identifying the input features for percolator

### ISSUE THREAD

**Issue by katrina-uw** [OpenMS/OpenMS#6693]
I'm trying to find out how to map these features back to the percolator:
![percolator_features](https://user-images.githubusercontent.com/45520179/218236761-58d3f9ea-3129-4363-a658-eb11dce2ed74.png)

For example, the PSM outputs from the percolator adaptor are shown below:
![psm_features](https://user-images.githubusercontent.com/45520179/218236824-c0ff69c7-2e75-42e1-892b-ec562eeda039.png)
There are many features above where the name is some id. I don't know the meaning behind since there are no descriptions..

Could you help me to check whether the features in the first figure can be found in the PSM output file (second figure), and what's the id of these features?

Thank you so much!

**Comment by timosachsenberg** [comment]
Hi 
The set of features differ between search engines and sometimes even search engine versions. So don’t expect the exact features add in the publication.

you can look up the meaning of the cv terms here https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo . 

In your input data the features written by the search engine are the ones that start with COMET.

**Comment by timosachsenberg** [comment]
Feel free to join our discord channel to discuss with other developers what you want to achieve with OpenMS https://discord.gg/4TAGhqJ7s5 .
Sometimes there might already been scripts etc. out there by others that could simplify things for you and people are usually happy to share.

**Comment by jpfeuffer** [comment]
Yes the MS:XXXXXX scores are usually coming from Percolator itself (i.e. output). You can check them in the ontology.
The input features specific for Comet are defined here:
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/ANALYSIS/ID/PercolatorFeatureSetHelper.cpp#L179

No matter which search engine also basic features like m/z, RT, charge are added, IIRC.

I think they are pretty stable but as Timo said, no guarantee that they were/will be all the same for older/newer versions.
